scripts/menu.js

```diff
@@ -0,0 +1,18 @@
+const menuBtn = document.querySelector('.menu');
+const mobileMenu = document.querySelector('.mobile-menu');
+
+function toggleMenu() {
+  if (menuBtn.classList.contains('active')) {
+    document.body.style.overflow = 'auto';
+  } else {
+    document.body.style.overflow = 'hidden';
+  }
+  menuBtn.classList.toggle('active');
+  mobileMenu.classList.toggle('show');
+}
```

Comment:
- [ ] [OPTONAL] Please kindly try to always use ES6 syntax.
```suggestion
const toggleMenu = () => {
  if (menuBtn.classList.contains('active')) {
    document.body.style.overflow = 'auto';
  } else {
    document.body.style.overflow = 'hidden';
  }
  menuBtn.classList.toggle('active');
  mobileMenu.classList.toggle('show');
}
```